fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/shim/NettyGrpcServer.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.fabric.shim;
+
+
+import io.grpc.Server;
+import io.grpc.netty.NettyServerBuilder;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * implementation grpc server with NettyGrpcServer.
+ */
+public class NettyGrpcServer implements GrpcServer {
+
+    private static final int MAX_INBOUND_METADATA_SIZE = 100 * 1024 * 1024;
```

Comment:
are these configurable from a properties file?

Reply:
in this version parameters not from properties file

Reply:
We should certainly make this configurable from the start.